instrumentation/github.com/gorilla/mux/otelmux/go.mod

```diff
@@ -11,11 +11,18 @@ require (
 	go.opentelemetry.io/otel/trace v1.35.0
 )
 
+require (
+	github.com/google/uuid v1.6.0 // indirect
+	go.opentelemetry.io/otel/sdk v1.35.0 // indirect
```

Comment:
@pellared is this going to be considered as a testing dependency? Will it not be imported by users of this instrumentation?